Makefile

```diff
@@ -2,7 +2,7 @@ DATA_DIR    = data
 
 DATA_FILES  = $(DATA_DIR)/templates/dir_listing.html
 DATA_FILES += $(DATA_DIR)/version_hash
-DATA_FILES += $(DATA_DIR)/static/js/jquery.tablesorter.min.js
+DATA_FILES += $(DATA_DIR)/static/js/jquery.tablesorter.combined.min.js
```

Comment:
This is the latest version of TableSorter built with its extensions saveSort and sort2Hash
I meant to add a note somewhere, but didn't (and not sure where would be best to)
https://github.com/Mottie/tablesorter/wiki/Customize
Those are the docs on how to minifiy/uglify a version of tablesorter with specific extensions/widgets
I didn't want to include all the widgets that we are not using that tablesorter has